Add section 2.4 (#12)
jneidel/internet.org-entwicklung: main.tex

\documentclass{article}
\usepackage[backend=biber]{biblatex}
\addbibresource{bibliography.bib}

\title{Wie entwickelte sich Facebooks Internet.org/Free Basics in Indien?}
\author{
  Fischer, Sarah Susanne\\
  \and
  Hein, Oliver\\
  \and
  Neidel, Jonathan\\
  \and
  Petersen, Tom Magnus\\
  \and
  Schmitz, Kai-Ibe\\
}

\begin{document}

\maketitle

% 2.3 in der Gliederung
\subsection{Rebranding als Free Basics}

> Drei Monate nach dem Start von Internet.org begann die indische Regierung unter Premierminister Narendra Modi die "Digital India campaign" (eng.: "Digitales Indien Kampagne") mit Zielen wie erweiterter Breitbandkonnektivität, Universalzugang zu Mobilnetzen, und einem Programm für öffentlichen Internetzugang. %cite: digitalindia.gov.in/content/programme-pillars

Reviewer: ```suggestion
Drei Monate nach dem Start von Internet.org begann die indische Regierung unter Premierminister Narendra Modi die `Digital India campaign' mit Zielen wie erweiterter Breitbandkonnektivität, Universalzugang zu Mobilnetzen, und einem Programm für öffentlichen Internetzugang. %cite: digitalindia.gov.in/content/programme-pillars
```

Eigenname, muss nicht übersetzt werden.

> Hierfür trat die Regierung mit Firmen des Silicon Valley in Kooperation, unter Anderem mit Facebook (Bereitstellung von Wifi Hotspots in ländlichen Gebieten).%cite everyday 254

Reviewer: ```suggestion
Hierfür trat die Regierung mit Firmen des Silicon Valley in Kooperation, unter anderem Facebook (Bereitstellung von Wifi Hotspots in ländlichen Gebieten) \parencite[254]{everydayLife},
```

> , wobei Modi und Zuckerberg ein öffentliches Image der engeren Zusammenarbeit aufbauten.

Reviewer: ```suggestion
wobei Modi und Zuckerberg ein öffentliches Image der engeren Zusammenarbeit aufbauten.
```

\medskip

> Dies führte selbstverständlich zu weiteren Unruhen unter den wachsenden Protesten gegen Internet.org, welche sich von nur wenigen Teilnehmern Anfang April 2015 zu global verteilten Kommentaren zuwider Facebook eineinhalb Monate später ausweiteten. %cite bbc.com/news/technology-32795270

Reviewer: ```suggestion
Dies führte selbstverständlich zu weiteren Unruhen unter den wachsenden Protesten gegen Internet.org, welche sich von nur wenigen Teilnehmern Anfang April zu global verteilten Kommentaren zuwider Facebook eineinhalb Monate später ausweiteten \autocite{BBC3}.
```

> Während diese primär online abliefen, fingen Menschen (zumeist in Indien) an, auf die Straße zu gehen.

Reviewer: ```suggestion
Während diese primär online abliefen, fingen Menschen an auf die Straße zu gehen.
```

Es ist davon auszugehen, das die Proteste in Indien stattfinden.

\medskip

> Zuckerbergs reaktive Äußerung, "universal connectivity and net neutrality can and must co-exist" ("universelle Konnektivität und Netzneutralität kann und muss koexistieren"), wobei der Dienst erstmalig als "basic free services" bezeichnet wurde, ließen die Stimmen (der "SafeTheInternet coalition") lediglich lauter erklingen, wenige Wochen später unterschrieben 67 sog. "digital rights" Aktivistengruppen (u.A. "i Freedom Uganda, "Usarios Digitales" aus Equador, und "ICT Watch" aus Indosnesien) einen an Facebook gerichteten Brief der Bedenken am Projekt.

Reviewer: ```suggestion
Zuckerbergs reaktive Äußerung ``universal connectivity and net neutrality can and must co-exist", wobei der Dienst erstmalig als ``basic free services" bezeichnet wurde, ließen die Stimmen (der `Safe The Internet coalition') lediglich lauter erklingen.
Wenige Wochen später unterschrieben 67 sog. `digital rights' Aktivistengruppen (u.A. `i Freedom Uganda', `Usarios Digitales' aus Equador und `ICT Watch' aus Indosnesien) einen an Facebook gerichteten Brief über Bedenken am Projekt.
```

Reviewer: Zitate habe ich in double quotes,
Eigennamen/begriffe in single quotes.

Reviewer: Englisch muss nicht übersetzt werden.

\medskip
\medskip

> Nachdem weitere Unternehmen aufgrund der Proteste austraten, gab Facebook den größten Stimmen nach. Unter dem neuen Namen "Free Basics" lockerte Facebook die Vorgaben für Entwickler, welche unter Anderem einen Vorbehalt sicherer Netzprotokolle beinhaltete, was zu einer Erweiterung des Angebots um 60 neue Seiten (z.B. Wikipedia) führte.<

Reviewer: ```suggestion
Nachdem weitere Unternehmen aufgrund der Proteste austraten, gab Facebook den größten Stimmen nach. Unter dem neuen Namen `Free Basics' lockerte Facebook die Vorgaben für Webseiten Betreiber und baute HTTPS Unterstützung ein, was zu einer Erweiterung des Angebots um 60 neue Seiten (z.B. Wikipedia) führte.
```
